test/e2e/set/scyllacluster/multi_datacenter_scyllacluster_external_seeds.go

```diff
@@ -0,0 +1,340 @@
+		framework.By("Waiting for the first ScyllaCluster to rollout (RV=%s)", sc1.ResourceVersion)
+		waitCtx1, waitCtx1Cancel := utils.ContextForRollout(ctx, sc1)
+		defer waitCtx1Cancel()
+		sc1, err = controllerhelpers.WaitForScyllaClusterState(waitCtx1, ns1Client.ScyllaClient().ScyllaV1().ScyllaClusters(sc1.Namespace), sc1.Name, controllerhelpers.WaitForStateOptions{}, utils.IsScyllaClusterRolledOut)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		verifyScyllaCluster(ctx, ns1Client.KubeClient(), sc1)
+		waitForFullQuorum(ctx, ns1Client.KubeClient().CoreV1(), sc1)
+
+		hosts1, hostIDs1, err := utils.GetBroadcastRPCAddressesAndUUIDs(ctx, ns1Client.KubeClient().CoreV1(), sc1)
+		o.Expect(err).NotTo(o.HaveOccurred())
+		o.Expect(hosts1).To(o.HaveLen(int(utils.GetMemberCount(sc1))))
+		o.Expect(hostIDs1).To(o.HaveLen(int(utils.GetMemberCount(sc1))))
+
+		di1 := insertAndVerifyCQLData(ctx, hosts1)
+		defer di1.Close()
+
+		seeds1, err := utils.GetBroadcastAddresses(ctx, ns1Client.KubeClient().CoreV1(), sc1)
+		o.Expect(err).NotTo(o.HaveOccurred())
+		o.Expect(seeds1).To(o.HaveLen(int(utils.GetMemberCount(sc1))))
+
+		sc2 := f.GetDefaultScyllaCluster()
+		sc2.Name = "multi-datacenter-cluster"
+		sc2.Spec.Datacenter.Name = "dc2"
+		sc2.Spec.Datacenter.Racks = []scyllav1.RackSpec{
+			{
+				Name:    "a",
+				Members: 1,
+				Storage: scyllav1.Storage{
+					Capacity: "1Gi",
+				},
+				Resources: corev1.ResourceRequirements{
+					Limits: corev1.ResourceList{
+						corev1.ResourceCPU:    resource.MustParse("1"),
+						corev1.ResourceMemory: resource.MustParse("1Gi"),
+					},
+					Requests: corev1.ResourceList{
+						corev1.ResourceCPU:    resource.MustParse("10m"),
+						corev1.ResourceMemory: resource.MustParse("100Mi"),
+					},
+				},
+			},
+			{
+				Name:    "b",
+				Members: 1,
+				Storage: scyllav1.Storage{
+					Capacity: "1Gi",
+				},
+				Resources: corev1.ResourceRequirements{
+					Limits: corev1.ResourceList{
+						corev1.ResourceCPU:    resource.MustParse("1"),
+						corev1.ResourceMemory: resource.MustParse("1Gi"),
+					},
+					Requests: corev1.ResourceList{
+						corev1.ResourceCPU:    resource.MustParse("10m"),
+						corev1.ResourceMemory: resource.MustParse("100Mi"),
+					},
+				},
+			},
+			{
+				Name:    "c",
+				Members: 1,
+				Storage: scyllav1.Storage{
+					Capacity: "1Gi",
+				},
+				Resources: corev1.ResourceRequirements{
+					Limits: corev1.ResourceList{
+						corev1.ResourceCPU:    resource.MustParse("1"),
+						corev1.ResourceMemory: resource.MustParse("1Gi"),
+					},
+					Requests: corev1.ResourceList{
+						corev1.ResourceCPU:    resource.MustParse("10m"),
+						corev1.ResourceMemory: resource.MustParse("100Mi"),
+					},
+				},
+			},
+		}
+		sc2.Spec.ExternalSeeds = append(make([]string, 0, len(seeds1)), seeds1...)
+		sc2.Spec.Sysctls = []string{"fs.aio-max-nr=30000000"}
+
+		framework.By("Creating namespace in the second cluster")
+		ns2, ns2Client := f.Cluster(1).CreateUserNamespace(ctx)
```

Comment:
nit: consistent indexing would make the logs more readable and less error prone

I suppose the fix with lowest changes and still using the "first" and "second" naming would be to do `cluster1=f.Cluster(0)` initially. I don't feel strong either way, even passing on this.

Reply:
I think programmers are pretty much used to the discrepancy between indexes and the names for elements in natural languages? You usually call `array[0]` the first element of the array, not the zero-th one. And we have
`ns1, ns1Client, ok := f.Cluster(0).DefaultNamespaceIfAny()` at the very beginning.

Reply:
I can live with that, it just looked a bit weird next to each other. I'd probably ten to call it `cluster #0` and avoid the skew.

Reply:
I can change it to `cluster #0`, mostly because it aligns with the artifacts better then.